sources/Outputs/include/Par.h

```diff
@@ -102,7 +108,7 @@ class Par {
    *
    * @returns nullptr if the generator use a SignalN model, the parameter set if not
    */
-  static boost::shared_ptr<parameters::ParametersSet> writeGenerator(const algo::GeneratorDefinition& def, const std::string& basename,
+  boost::shared_ptr<parameters::ParametersSet> writeGenerator(const algo::GeneratorDefinition& def, const std::string& basename,
```

Comment:
Keep the methods static and put the active power compensation in argument. The rational behind the 'static' pattern was to only use the required member to build the part of the file, allowing to better understand which part is using which data.